modules/rostests/apitests/crt/crtdata.c

```diff
@@ -609,6 +646,34 @@ START_TEST(crtdata)
     s_osvi.dwOSVersionInfoSize = sizeof(OSVERSIONINFOW);
     GetVersionExW(&s_osvi);
 
+    HMODULE hmsvcrt = GetModuleHandleA("msvcrt.dll");
+    p__daylight = (void*)GetProcAddress(hmsvcrt, "__daylight");
+    p__dstbias = (void*)GetProcAddress(hmsvcrt, "__dstbias");
+    p_get_fmode = (void*)GetProcAddress(hmsvcrt, "_get_fmode");
+    p_get_osplatform = (void*)GetProcAddress(hmsvcrt, "_get_osplatform");
+    p_get_pgmptr = (void*)GetProcAddress(hmsvcrt, "_get_pgmptr");
+    p_get_wpgmptr = (void*)GetProcAddress(hmsvcrt, "_get_wpgmptr");
+    p_get_winver = (void*)GetProcAddress(hmsvcrt, "_get_winver");
+    p_get_winmajor = (void*)GetProcAddress(hmsvcrt, "_get_winmajor");
+    p_get_winminor = (void*)GetProcAddress(hmsvcrt, "_get_winminor");
+    p_get_environ = (void*)GetProcAddress(hmsvcrt, "_get_environ");
+    p_get_wenviron = (void*)GetProcAddress(hmsvcrt, "_get_wenviron");
+
+    if (s_osvi.dwMajorVersion >= 6)
```

Comment:
Why don't use OS version helper functions?

Reply:
This was already used?

Reply:
The best would probably to use the PEB field, because IIRC that is the only way to get the real version, since WIndows tends to lie. But for now this is good enough.